### DISCUSSION TITLE
qpdfviewer - toolbar

### DISCUSSION THREAD

**Discussion by lukejohnt** [quasarframework/quasar#12406] [Quasar App Extensions] [answered]
Is there a way to hide some of the tools on the toolbar. i.e. We don't want the user to be able to print or download a document just view only.
Thanks

**Comment by hawkeye64** [comment, marked as answer]
At this time, there is no way of doing this. You would have to load in your own PDFjs instance and control it that way. If you are just building for a web browser, you can instantiate the native PDF Viewer and control it yourself as well.
